src/content/release-notes.mdx

```diff
@@ -5,6 +5,39 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.23.0
+
+12 August 2024
+
+This versions is compatible with Kubernetes versions 1.26 to 1.30
+
+### New Features
+
+- Added a [feature flag to enable/disable autowake](self-hosted/helm-configuration.mdx#autowake) for Namespaces on incoming request  <!-- 8379 -->
+- Support for GitLab Subgroups and Groups with nested Subgroups <!-- 8398, 8393, 8394 -->
+- Added support for Kubernetes 1.30  <!-- 8421 -->
+
+### Improvements
+
+- Prevent the usage of reserved variable names  <!-- 8272 -->
+- Fix for git-urls CVE-2023-46402  <!-- 8300 -->
+- Updated registry version to 2.8.2-75b5eee  <!-- 8368 -->
+- Reconcile selected resource state consistently with URL  <!-- 8363 -->
+- Redesigned the `Setting` Dashboard component to show status (e.g. Garbage Collector installed/not installed)  <!-- 8380 -->
+
+### Bug Fixes
+
+- For administrators, the Dashboard now shows the admin alert banner when looking at other users' empty Namespaces and destroy logs  <!-- 8293 -->
+- Disabled the delete button for any Personal Namespace  <!-- 8295 -->
+- Inject the okteto-cluster-admin service account into installer jobs and grant registry access to all admins  <!-- 8299 -->
+- Fix to gracefully handle localStorage errors  <!-- 8310 -->
+- Fixed frontend user roles and permissions regressions  <!-- 8314 -->
+- Ignore Okteto role bindings during validation when the flag is enabled  <!-- 8343 -->
+- Fixed layout shifting upon loading staged logs  <!-- 8406 -->
+- Fix "variables ignored" warning shown on deploy-URL for "Develop On Okteto Button" feature despite not having specified any variables  <!-- 8414 -->
+- Bumped daemon to 0.13.0 and cli to 2.30.0-beta.2  <!-- 8411 -->
+
```

Comment:
We have a few items from the CLI which are relevant to include. I will mention them in this comment indicating also the category:

* **Improvement**: You can now Build images using `okteto build` command as part of your custom commands in the Okteto manifest when executing a remote deploy/destroy. Previously to this, no image was being built
* **Improvement**: You can now specify hosts to be resolve to specific IPs for your tests defined under `test` section in the Okteto Manifest. We might add a link to https://www.okteto.com/docs/reference/okteto-manifest/#hosts-string-optional
* **Fix**:  Fixed a bug that was preventing `okteto test` to export artifacts when a folder was specified under `artifacts` key

Reply:
Added!